assets/app/scripts/controllers/build.js

```diff
@@ -99,8 +99,34 @@ angular.module('openshiftConsole')
           if (!$filter('isIncompleteBuild')(build)){
             delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
           }
-        }        
+        }
       }));
+
+      (function initLogs() {
```

Comment:
Curious about the immediately invoked function because I don't see any vars declared. Is this just to give the block a descriptive name?